tomopy/misc/npmath.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import numpy  as np
+import logging
+import warnings
+
+logger = logging.getLogger(__name__)
+
+
+__author__ = "Chen Zhang"
+__all__    = [ 'gauss',
+               'calc_cummulative_dist',
+               'calc_affine_transform',
+]
+
+
+def gauss(x, *p):
+    """
+    simple Guassian func used for quick curve fit
+
+    Parameters
+    ----------
+    x  :  np.1ndarray
+        1D array for curve fitting
+    p  :  parameter lis t
+        A,          mu,      sigma  =   p
+        magnitude  center     std
+
+    Returns
+    -------
+    1d Gaussian distrobution evaluted at x with p
+    """
+    A, mu, sigma = p
+    return A*np.exp(-(x-mu)**2/(2.*sigma**2))
+
+
+def calc_cummulative_dist(data, steps=None):
+    """
+    Calculated the cumulative distribution from a data array without using
```

Comment:
Can you add a sentence about what you refer to by saying binning?

Calculated -> Calculate
using can be dropped.
This is a more robust way of representing